test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs

```diff
@@ -836,45 +837,90 @@ public class MNISTPrediction
         {
             var modelLocation = "cifar_model/frozen_model.pb";
-
             var mlContext = new MLContext(seed: 1);
+            List<string> logMessages = new List<string>();
+            mlContext.Log += (sender, e) => logMessages.Add(e.Message);
             var tensorFlowModel = mlContext.Model.LoadTensorFlowModel(modelLocation);
             var schema = tensorFlowModel.GetInputSchema();
             Assert.True(schema.TryGetColumnIndex("Input", out int column));
-            var type = (VectorDataViewType)schema[column].Type;
+            var type = (VectorDataViewType) schema[column].Type;
             var imageHeight = type.Dimensions[0];
             var imageWidth = type.Dimensions[1];
 
             var dataFile = GetDataPath("images/imagesmixedpixelformat.tsv");
             var imageFolder = Path.GetDirectoryName(dataFile);
             var data = mlContext.Data.LoadFromTextFile(dataFile,
                 columns: new[]
-                    {
-                        new TextLoader.Column("ImagePath", DataKind.String, 0),
-                        new TextLoader.Column("Name", DataKind.String, 1),
-                    }
-                );
+                {
+                    new TextLoader.Column("ImagePath", DataKind.String, 0),
+                    new TextLoader.Column("Name", DataKind.String, 1),
+                }
+            );
 
-            var pipeEstimator = new ImageLoadingEstimator(mlContext, imageFolder, ("ImageReal", "ImagePath"))
-                .Append(new ImageResizingEstimator(mlContext, "ImageCropped", imageWidth, imageHeight, "ImageReal"))
-                .Append(new ImagePixelExtractingEstimator(mlContext, "Input", "ImageCropped", interleavePixelColors: true));
+            var pipeEstimator = new ImageLoadingEstimator(mlContext, imageFolder,
+                    ("ImageReal", "ImagePath"))
+                .Append(new ImageResizingEstimator(mlContext, "ImageCropped",
+                    imageWidth, imageHeight, "ImageReal"))
+                .Append(new ImagePixelExtractingEstimator(mlContext, "Input",
+                    "ImageCropped", interleavePixelColors: true));
 
             var pixels = pipeEstimator.Fit(data).Transform(data);
-            IDataView trans = tensorFlowModel.ScoreTensorFlowModel("Output", "Input").Fit(pixels).Transform(pixels);
+            IDataView trans = tensorFlowModel.ScoreTensorFlowModel("Output", "Input")
+                .Fit(pixels).Transform(pixels);
 
             trans.Schema.TryGetColumnIndex("Output", out int output);
             using (var cursor = trans.GetRowCursor(trans.Schema["Output"]))
             {
-                var buffer = default(VBuffer<float>);
-                var getter = cursor.GetGetter<VBuffer<float>>(trans.Schema["Output"]);
-                var numRows = 0;
-                while (cursor.MoveNext())
+                using (var cursor2 = trans.GetRowCursor(trans.Schema["Output"]))
                 {
```

Comment:
nit: Common formatting pattern in such cases should  reduce the indentation and make it more readable
            using (var cursor = trans.GetRowCursor(trans.Schema["Output"]))
            using (var cursor2 = trans.GetRowCursor(trans.Schema["Output"]))
            {
                var buffer = defa

Reply:
ah nice thanks 

---
In reply to: [280192257](https://github.com/dotnet/machinelearning/pull/3625#discussion_r280192257) [](ancestors = 280192257)